README.md

```diff
@@ -90,49 +90,43 @@ With tg-resources, all errors are Rejected. The logic is best described with an
 ```js
 const resource = new Resource('user/login');
 
+const errorHandler = (error) => {
+    // Network error occurred
+    if (error.isNetworkError) {
+        console.error({
+            type: 'NETWORK_FAILED',
+            error,
+        });
+    } else if (error.isValidationError) {
```

Comment:
Would like to have example about ValidationErrorInterface here (or in the examples folder).